sycl/include/sycl/interop_handle.hpp

```diff
@@ -134,6 +139,26 @@ class interop_handle {
 #endif
   }
 
+  using graph = ext::oneapi::experimental::command_graph<
+      ext::oneapi::experimental::graph_state::executable>;
+  template <backend Backend = backend::opencl>
+  backend_return_t<Backend, graph> ext_codeplay_get_native_graph() const {
```

Comment:
`interop_handle` is also usable with `host_task`, as I understand it should fail `ext_codeplay_has_graph()` in a host-task, but should we document in our spec that this will happen?

Reply:
Indeed, and I wrote a test that asserts that.

This behavior was implicit though, and maybe borders on implementation detail, so I added a sentence to the spec description of this query.